### PR TITLE
convert module/classes names to lower case

### PR DESCRIPTION
**Pull request by harshithad0703**
Changes:

- Stack -> stack
- ContentType -> contentType
- Entry -> entry
- Asset -> asset
- ImageTransform -> imageTransform

### CODE REVIEW

File: src/lib/image-transform.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ import {
   TransformData,
 } from './types';
 
-export class ImageTransform {
```
Comment (nadeem-cs) [inline src/lib/image-transform.ts:15]: @harshithad0703 No need to change for ImageTransform class. As we're directly using this class, better to use Pascal case here

File: test/api/entry-queryables.spec.ts [hunk 2]
```diff
@@ -243,9 +243,17 @@ describe('Query Operators API test cases', () => {
         expect(query.entries[0].created_at).toBeDefined();
       }
     });
+
+    it('should check for include reference', async () => {
+      const query = makeEntries('contenttype_uid').includeReference('abc').query()
+      const result = await query.find<TEntry>()
+      if (query) {
+      console.log("🚀 ~ it ~ query._p:", query._queryParams)
```
Comment (nadeem-cs) [inline test/api/entry-queryables.spec.ts:251]: @harshithad0703 Remove console.log and add proper assertions

**Review by nadeem-cs — CHANGES_REQUESTED** [review]

File: src/lib/image-transform.ts (hunk 1 above)
Comment (harshithad0703) [reply src/lib/image-transform.ts]: made changes

File: test/api/entry-queryables.spec.ts (hunk 2 above)
Comment (harshithad0703) [reply test/api/entry-queryables.spec.ts]: added

**Review by nadeem-cs — APPROVED** [review]
Approved.

**Pull request merged**